pyanaconda/core/constants.py

```diff
@@ -397,6 +397,7 @@ class DisplayModes(Enum):
 PAYLOAD_TYPE_LIVE_OS = "LIVE_OS"
 PAYLOAD_TYPE_LIVE_IMAGE = "LIVE_IMAGE"
 PAYLOAD_TYPE_RPM_OSTREE = "RPM_OSTREE"
+PAYLOAD_TYPE_RPM_BOOTC = "RPM_BOOTC"
```

Comment:
Just a `bootc` please. The `RPM_OSTREE` is based on the name of the tool which is called `rpm-ostree` so to keep the same level of consistency we should call it just `bootc`.

Reply:
Done